src/cbmc/cbmc_parse_options.cpp

```diff
@@ -310,6 +311,11 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
     options.set_option("refine-arithmetic", true);
   }
 
+  if(cmdline.isset("string-refine"))
```

Comment:
It would be more coherent with the naming of the existing options to call this `refine-strings`